src/app/legacy/psammead/psammead-image/src/index.jsx

```diff
@@ -65,7 +64,8 @@ export const Img = props => {
           <StyledImg src={src} {...otherProps} />
         </StyledPicture>
       )}
-      {pageType !== FRONT_PAGE && (
+
+      {[FRONT_PAGE, HOME_PAGE].includes(pageType) && (
```

Comment:
Should this be the inverse (based on the diff)
```suggestion
      {![FRONT_PAGE, HOME_PAGE].includes(pageType) && (
```